subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -155,7 +155,7 @@ static void send_pending_adv(struct k_work *work)
 
 	atomic_clear_bit(adv.flags, ADV_FLAG_SCHEDULED);
 
-	while ((buf = net_buf_get(&bt_mesh_adv_queue, K_NO_WAIT))) {
+	while ((buf = bt_mesh_adv_buf_get(K_NO_WAIT))) {
 		/* busy == 0 means this was canceled */
 		if (!BT_MESH_ADV(buf)->busy) {
```

Comment:
This `busy` logic can be moved inside `bt_mesh_adv_buf_get` now

Reply:
Compared with the modification of existing logic and keep assist with legacy with ext, it has little significance.